Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesProvider.kt

```diff
@@ -38,29 +40,27 @@ class PersonCertificatesProvider @Inject constructor(
         dccWalletInfoRepository.personWallets
     ) { certificateContainer, cwaUser, personWallets ->
 
-        val allCerts = certificateContainer.allCwaCertificates
         val vaccPersons = certificateContainer.vaccinationInfos
-
         val personWalletsGroup = personWallets.associateBy { it.personGroupKey }
+        val allCerts = certificateContainer.allCwaCertificates.groupByPerson()
```

Comment:
Maybe call it `groupedCerts` instead of `allCerts` to make it clear that they are already grouped?

Reply:
renamed